smallsetup/StartK8s.ps1

```diff
@@ -423,6 +423,8 @@ route delete $clusterCIDRServicesWindows >$null 2>&1
 Write-Log "Add route to $clusterCIDRServicesWindows"
 route -p add $clusterCIDRServicesWindows $ipControlPlane METRIC 7 | Out-Null
 
+Restart-NlaSvc
```

Comment:
Can we do this when HNS, vmcompute services are restarted? Is there a particular significance doing it here? Same goes to Start script as well.